h/static/scripts/directives/annotation.coffee

```diff
@@ -52,6 +53,15 @@ AnnotationController = [
 
     ###*
     # @ngdoc method
+    # @name annotation.AnnotationController#tagsAutoComplete.
+    # @returns {Promise} immediately resolved to {string[]} -
+    # the tags to show in autocomplete.
+    ###
+    this.tagsAutoComplete = (query) ->
+      $.when(tagHelpers.filterTags(query))
```

Comment:
Because I think that's also a jquery function, but it's better if we don't depend on jquery and just use what angular gives us.